src/vs/workbench/contrib/workspace/browser/workspace.contribution.ts

```diff
@@ -438,13 +438,6 @@ export class WorkspaceTrustUXHandler extends Disposable implements IWorkbenchCon
 
 		if (openFiles.length) {
 			this.showIndicatorsInEmptyWindow = true;
-
-			// If all open files are trusted, transition to a trusted workspace
-			const openFilesTrustInfo = await Promise.all(openFiles.map(uri => this.workspaceTrustManagementService.getUriTrustInfo(uri!)));
-
-			if (openFilesTrustInfo.map(info => info.trusted).every(trusted => trusted)) {
-				this.workspaceTrustManagementService.setWorkspaceTrust(true);
-			}
 		} else {
```

Comment:
as discussed, I think we can just remove this altogether since the default setting is trusted. this will really simplify the contribution.